src/cc/frontends/clang/libbpf.h

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (c) 2015 PLUMgrid, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eBPF mini library */
+
+#ifndef LIBBPF_H
+#define LIBBPF_H
+
+#include "compat/linux/bpf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef int (*bpf_create_map_cb_t)(void *cb_cookie, void *data);
```

Comment:
So this wont fly at all, it was done for prototyping but there's no reason to duplicate entire libbpf.h , upstream wont accept it. Find a way to reuse the existing libbpf.h